### PR TITLE
Update Azure DevOps docs with RSA SHA-2 keys

### PR DESCRIPTION
**Pull request by stefanprodan**
Fix: https://github.com/fluxcd/flux2/issues/4726

DevOps doc preview: https://deploy-preview-1936--fluxcd.netlify.app/flux/installation/bootstrap/azure-devops/

### CODE REVIEW

File: content/en/flux/installation/bootstrap/azure-devops.md [hunk 1]
```diff
@@ -51,29 +51,28 @@ inside the `flux-system` namespace.
...
-flux bootstrap git \
-  --url=https://dev.azure.com/<org>/<project>/_git/<repository> \
-  --branch=main \
-  --password=${GIT_PASSWORD} \
-  --ssh-hostname=ssh.dev.azure.com \
-  --ssh-key-algorithm=rsa \
-  --ssh-rsa-bits=4096 \
-  --path=clusters/my-cluster
-```
+Azure DevOps SSH works only with RSA SHA-2 keys, to generate a key pair run:
```
Comment (souleb) [inline content/en/flux/installation/bootstrap/azure-devops.md:66]:
```suggestion
Azure DevOps SSH works only with RSA SHA-2 keys. To generate a key pair run:
```

**Review by souleb — APPROVED** [review]
LGTM!

File: content/en/flux/installation/bootstrap/azure-devops.md [hunk 2]
```diff
@@ -51,29 +51,29 @@ inside the `flux-system` namespace.
...
 Note that Azure DevOps PAT have an expiry date. To rotate the token before it expires,
-delete the `flux-system` secret from the cluster and re-run
-the bootstrap command using a valid PAT.
+delete the `flux-system` secret from the cluster and create a new one with the new PAT:
+
+```sh
+flux create secret git flux-system \
+   --url=https://dev.azure.com/<org>/<project>/_git/<repository> \
+   --username=git \
+   --password=$AZURE_DEVOPS_PAT
```
Comment (makkes) [inline content/en/flux/installation/bootstrap/azure-devops.md:60]: This environment variable hasn't been used prior on this page but `<az-token>` so maybe use that for consistency's sake?
```suggestion
   --password=<az-token>
```

**Review by makkes — COMMENTED** [review]
just a minor comment, otherwise lgtm

**Pull request merged**